grice/db_controller.py

```diff
@@ -158,6 +158,23 @@ def parse_join(join_str, outer_join: bool):
     return TableJoin(table_name, column_pairs, outer_join)
 
 
+def parse_col_names(column_names):
+    """
+    This method takes a string of comma-seperated column names and returns a list of column names.
+
+    :param column_names: string
+    :return: column_names: list
+    """
+    if column_names:
+        columns_dict = OrderedDict()
```

Comment:
I think I copy and pasted your code for parse_sorts for this... :) I was going to ask you why you were doing it this way... but yeah, perhaps just `set(column_names.split(',)` is all that is really required?